DTS-STN/Alpha-Site
# Splash page fix and br removal

# Description

No task on the Trello board. I removed all the br in index page. I also centered the text on the splash page button, it wasn't centered on main.

Splash page on main
![image](https://user-images.githubusercontent.com/72703030/120515773-a70e9f80-c39c-11eb-9723-33abc0550cf3.png)

Splash page fix
![image](https://user-images.githubusercontent.com/72703030/120515678-89d9d100-c39c-11eb-97ba-5825bbcc0218.png)

Index page
![image](https://user-images.githubusercontent.com/72703030/120515906-cd343f80-c39c-11eb-92c4-fefd69c47345.png) 

## Acceptance Criteria

There shouldn't be any br in the index page.
Splash page should have centered buttons.

## Test Instructions

1. Check buttons on splash page
2. Check code for index page, there shouldn't be any br and page paragraphs on the page should have spacing between.

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
